optimum/exporters/onnx/model_configs.py

```diff
@@ -77,11 +78,13 @@ class Seq2SeqDecoderOnnxConfig(TextSeq2SeqOnnxConfig):
 
     @property
     def inputs(self) -> Mapping[str, Mapping[int, str]]:
-        common_inputs = {
-            "decoder_input_ids": {0: "batch_size", 1: "past_decoder_sequence_length + sequence_length"},
-            "encoder_outputs": {0: "batch_size", 1: "encoder_sequence_length"},
-            "attention_mask": {0: "batch_size", 1: "encoder_sequence_length"},
-        }
+        common_inputs = OrderedDict(
```

Comment:
Not in favor of that, especially because we can get it to work without it.

Reply:
Okay, I will remove.